mpy-cross/Makefile

```diff
@@ -11,11 +11,13 @@ override undefine BUILD
 override undefine PROG
 endif
 
-include ../py/mkenv.mk
-
-# define main target
+# PROG is used in py/mkenv.mk. Define first.
+# define main target. Compile to run on Host (Linux, OSX or Windows),
+# do not include .exe in PROG.
```

Comment:
This is sort of obvious since the line below doesn't have the .exe ?

Reply:
I want to document the rule somewhere. It affects whether to cross or native compile. The rules are:
1. To always compile for Windows target (eg micropython port) add .exe
2. To compile for the Host environment (eg mpy-cross) don't add .exe.

Is there a better way / place to document this?

Reply:
Ah now I understand; this is probably the best place to document it, but I only understood what you mean after your comment here. So I'd suggest to reword the comment in the Makefile to make these rules clear.